mods/stairs/init.lua

```diff
@@ -804,3 +804,22 @@ stairs.register_stair_and_slab(
 	"Snow Block Slab",
 	default.node_sound_snow_defaults()
 )
+stairs.register_stair_and_slab(
+	"glass",
+	"default:glass",
+	{cracky = 3, oddly_breakable_by_hand = 3},
+	{"default_glass.png", "default_glass.png", "stairs_glass_slab.png"},
+    "Glass Stair",
```

Comment:
Indentation

Reply:
yeh, i forgot when using text editor I have to use 2 tabs, will fix